src/api_functions.py

```diff
@@ -19,11 +19,11 @@ def hyp3_login() -> API:
 
     username = ""
     try:
-        f = open('.netrc', 'r')
-        contents = f.read()
+        with open('.netrc', 'r')as f:
+            contents = f.read()
         username = contents.split(' ')[3]
         password = contents.split(' ')[5].split('\n')[0]
-    except IndexError:
+    except IndexError and FileNotFoundError:
```

Comment:
Having an except block just to pass seems like dead code. 
Perhaps exit the program gracefully or send a warning message.